### PR TITLE
fix alert when email already exists from

### LINKED ISSUE #2375: Incorrect error message when user email already exists
If we try to signup using an existing email id, the message shown is - invalid email, while it should be like - email already exists. 

image

image

### PR DESCRIPTION
**Pull request by Sync271**
Fixes #2375 
When the email already exists I should say "Email already exists" instead of "Invalid email"

### CODE REVIEW

File: frontend/src/SignupPage/SignupPage.jsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ class SignupPage extends React.Component {
         this.setState({ isLoading: false, signupSuccess: true });
       },
       () => {
-        toast.error('Invalid email', {
+        toast.error('Email already exists', {
```
Comment (Navaneeth-pk) [inline frontend/src/SignupPage/SignupPage.jsx:45]: I think we need to display the error message in the response from server. The request can fail for many reasons.

Comment (Sync271) [reply frontend/src/SignupPage/SignupPage.jsx:45]: > I think we need to display the error message in the response from the server. The request can fail for many reasons.

Yeah, but how'd you check the error tho? I am not sure how to go about this, let me know if you have any suggestions

Comment (Navaneeth-pk) [reply frontend/src/SignupPage/SignupPage.jsx:45]: `server/src/services/auth.service.ts:61` - you might need to add some logic here to respond with a meaningful message if the email already exists.

**Comment by Navaneeth-pk** [comment]
Just noticed that this issue is already assigned to someone. Please create PRs only for the issues that are assigned to you. 
@Sync271

**Pull request closed without merging**